PWGCF/DataModel/FemtoDerived.h

```diff
@@ -61,6 +65,10 @@ DECLARE_SOA_TABLE_STAGED(FDCollisions, "FDCOLLISION",
                          femtodreamcollision::MagField);
 using FDCollision = FDCollisions::iterator;
 
+DECLARE_SOA_TABLE(FDExtQnCollisions, "AOD", "FDEXTCOLLISION",
```

Comment:
I guess this will not work when going to the derived data files
The rule, if I remember correctly, should be: the name of the table, `FDExtQnCollisions` should match the capitalized name in the AO2D file but without `s`. In this concrete case it should be `"FDEXTQNCOLLISION"`
I think this last name surpass the number of characters allowed for the tables on AO2D files. Please, check it and modify it accordingly